localisation/00_zuwc_united_localisation.csv

```diff
@@ -1541,35 +1531,35 @@ EVTDESC_WCVOD_2005;Your servants informed you that a humpbacked wrinkled old man
+EVTDESC_WCVOD_2010;After you took over that ritual blade, you began to hear voices. During that couple of days, while it was in your hands, you were ceaselessly having nightmares, which were cut off by the gentle whispers of Xal'atath. Have you lost your mind? Or is this blade cursed and tries to speak to you? The blade beckons you!;;Nachdem Ihr die Ritualklinge an Euch genommen hattet, fingt Ihr an, Stimmen zu h�ren. W�hrend der wenigen Tage, die sie in Euren H�nden war, hattet Ihr unaufh�rlich Alptr�ume, die durch das sanfte Fl�stern von Xal'atath unterbrochen wurden. Habt Ihr den Verstand verloren? Oder ist diese Klinge verflucht und versucht, mit Euch zu sprechen? Die Klinge ruft Euch!;;;;;;;;;;;x
 EVTOPTA_WCVOD_2010;Xal'atath is calling me!;;Xal'atath ruft mich!;;;;;;;;;;;x
 EVTOPTB_WCVOD_2010;Throw it out of here!;;Werft sie hinaus!;;;;;;;;;;;x
-EVTDESC_WCVOD_2015;One night, you, led by the whispers of Xal'atath, secretly crept into the treasury. There, a black blade told you about hundreds of dark secrets of Azeroth: in the farthest corners of this world there lives a true madness, a first-born evil. In ancient times, Xal'atath shed blood of the sea for the sake of this evil... but now these times are behind... Your determination amuses Xa'alath, he is ready to guide you... for now.;;Eines Nachts schlicht Ihr Euch, geleitet vom Fl�stern Xal'ataths, heimlich in die Schatzkammer. Dort erz�hlte Euch die schwarze Klinge von Hunderten dunkler Geheimnisse Azeroths: In den entlegensten Ecken dieser Welt herrscht der reinste Wahnsinn, ein erstgeborenes �bel. In alten Zeiten vergoss Xal'atath f�r dieses �bel Meere von Blut... aber jetzt sind diese Zeiten vorbei... Eure Entschlossenheit am�siert Xa'alath, er ist bereit, Euch zu f�hren... vorerst.;;;;;;;;;;;x
+EVTDESC_WCVOD_2015;One night, you secretly crept into the treasury, led by the whispers of Xal'atath. There, the black blade told you about hundreds of dark secrets of Azeroth: in the farthest corners of this world lives a true madness, a first-born evil. In ancient times, Xal'atath shed seas of blood in the name of dark elders... but now, these times are behind... Your determination amuses Xa'alath, she is ready to guide you... for now.;;Eines Nachts schlicht Ihr Euch, geleitet vom Fl�stern Xal'ataths, heimlich in die Schatzkammer. Dort erz�hlte Euch die schwarze Klinge von Hunderten dunkler Geheimnisse Azeroths: In den entlegensten Ecken dieser Welt herrscht der reinste Wahnsinn, ein erstgeborenes �bel. In alten Zeiten vergoss Xal'atath f�r dieses �bel Meere von Blut... aber jetzt sind diese Zeiten vorbei... Eure Entschlossenheit am�siert Xa'alath, sie ist bereit, Euch zu f�hren... vorerst.;;;;;;;;;;;x
```

Comment:
The text is still pretty broken. There's a typo in Xal'atath.
```suggestion
EVTDESC_WCVOD_2015;One night, you secretly crept into the treasury, led by the whispers of Xal'atath. In there, the black blade told you about hundreds of dark secrets of Azeroth: in the farthest corners of this world, a true madness lives, the first-born evil. In ancient times, Xal'atath shed seas of blood in the name of dark elders... but now, these times are behind. Your determination entertains Xal'atath, she is ready to guide you... for now.;;Eines Nachts schlicht Ihr Euch, geleitet vom Fl�stern Xal'ataths, heimlich in die Schatzkammer. Dort erz�hlte Euch die schwarze Klinge von Hunderten dunkler Geheimnisse Azeroths: In den entlegensten Ecken dieser Welt herrscht der reinste Wahnsinn, ein erstgeborenes �bel. In alten Zeiten vergoss Xal'atath f�r dieses �bel Meere von Blut... aber jetzt sind diese Zeiten vorbei... Eure Entschlossenheit am�siert Xa'alath, sie ist bereit, Euch zu f�hren... vorerst.;;;;;;;;;;;x
```

Reply:
done